File: Listy/Zad_08_01_Lista1Zad1EntryPoint/app/src/main/java/pl/edu/uwr/pum/lista5zad1entrypoint/MainActivity.java
package pl.edu.uwr.pum.lista5zad1entrypoint;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button button = findViewById(R.id.first);
        button.setOnClickListener(view -> {
            Intent intent = new Intent(this, SecondActivity.class);
            startActivity(intent);
        });
    }
}

File: Listy/Zad_08_01_Lista1Zad1EntryPoint KOTLIN/app/src/main/java/com/example/zad_08_01_lista1zad1entrypoint/MainActivity.kt
package com.example.zad_08_01_lista1zad1entrypoint

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button

class MainActivity : AppCompatActivity() {

    private lateinit var myButton: Button

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        myButton = findViewById(R.id.button)
        myButton.setOnClickListener {
            val intent: Intent = Intent(this, SecondActivity::class.java)
            startActivity(intent)
        }
    }
}